doc/design/upgrade.md

```diff
@@ -26,7 +45,9 @@ There will be impact on container interaction while the appliance is down:
 - no attach ability
 - network outage for NAT based port forwarding
 
-All of those facets should resume normal operation after upgrade is complete
+All of those facets should resume normal operation after upgrade is complete. 
```

Comment:
It should be and should be resume after VCH restart. But yeah, @hmahmood please confirm. thanks